spec/services/shopify_import/data_parsers/customers/base_data_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+
+RSpec.describe ShopifyImport::DataParsers::Customers::BaseData do
```

Comment:
What do you mean? Test that `created_at` is kind of a DateTime?

I just followed what was in product base data specs:
https://github.com/spark-solutions/spree_shopify_importer/blob/master/app/services/shopify_import/data_parsers/products/base_data.rb
https://github.com/spark-solutions/spree_shopify_importer/blob/master/app/services/shopify_import/data_parsers/products/base_data.rb